SPECS/selinux-policy/0038-enable-liveos-iso-flow.patch

```diff
@@ -131,6 +132,29 @@ index ea08036b6..59c1afeaf 100644
  allow kernel_t self:process { transition signal_perms getsched setsched getsession getpgid setpgid getcap setcap share getattr noatsecure siginh rlimitinh dyntransition setkeycreate setsockcreate getrlimit };
  allow kernel_t self:shm create_shm_perms;
  allow kernel_t self:sem create_sem_perms;
+diff --git a/policy/modules/services/ssh.te b/policy/modules/services/ssh.te
+index f3c966ed0..8075c1240 100644
+--- a/policy/modules/services/ssh.te
++++ b/policy/modules/services/ssh.te
+@@ -243,6 +243,8 @@ allow sshd_t self:key { search link write };
+ 
+ allow sshd_t sshd_keytab_t:file read_file_perms;
+ 
++allow sshd_t var_t:file { getattr open read };
++
+ manage_dirs_pattern(sshd_t, sshd_tmp_t, sshd_tmp_t)
+ manage_files_pattern(sshd_t, sshd_tmp_t, sshd_tmp_t)
+ manage_sock_files_pattern(sshd_t, sshd_tmp_t, sshd_tmp_t)
+@@ -343,6 +345,9 @@ allow ssh_keygen_t self:unix_stream_socket create_stream_socket_perms;
+ allow ssh_keygen_t sshd_key_t:file manage_file_perms;
+ files_etc_filetrans(ssh_keygen_t, sshd_key_t, file)
+ 
++allow ssh_keygen_t var_t:dir { write add_name remove_name };
++allow ssh_keygen_t var_t:file { getattr setattr create open read write rename };
```

Comment:
`var_t` typically isn't used for files, since it is extremely generic, policy-wise. What dir(s)/file(s) are created?

Reply:
Looked into it further, it turns out we're moving the ssh key file location from /etc/ to /var/etc:
```
/var/etc/ssh/ssh_host_rsa_key.pub
/var/etc/ssh/ssh_host_rsa_key
/var/etc/ssh/ssh_host_ecdsa_key.pub
/var/etc/ssh/ssh_host_ecdsa_key
/var/etc/ssh/ssh_host_ed25519_key.pub
/var/etc/ssh/ssh_host_ed25519_key
```

This change is necessary for when verity is enabled and the etc location is read-only with a volatile overlay.

I'll drop the policy changes from the PR based on that. Thanks Chris.